### PR TITLE
feat: add back member key in app endpoints

### LINKED ISSUE #1388: Provide back `member` for app endpoints
To continue to support legacy apps, we will still provide the `member` property.

Mainly because of Twente apps.

### PR DESCRIPTION
**Pull request by pyphilia**
I added back member to:

- POST /app-data (payload: memberId, return: member)
- PATCH /app-data (return)
- GET /app-data (payload)
- POST /app-action (return)
- GET /app-action (return)

I also removed GET many app-data, GET many app-action, I don't think we used it.

close #1388

### CODE REVIEW

File: src/services/item/plugins/app/appAction/schemas.ts [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
     appAction: {
       type: 'object',
       properties: {
         id: { type: 'string' },
-        member: {
-          $ref: 'https://graasp.org/members/#/definitions/member',
-        },
+        account: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
+        /** @deprecated use account */
+        member: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
```
Comment (spaenleh) [inline src/services/item/plugins/app/appAction/schemas.ts:10]: I think we should mark the JSONSchema as deprecated: https://stackoverflow.com/questions/49870109/how-to-annotate-a-field-as-deprecated-in-openapi-swagger-2-0
Sop it shows on OpenAPI documentation.

File: src/services/item/plugins/app/appAction/schemas.ts [hunk 2]
```diff
@@ -42,7 +39,9 @@ const getForOne = {
   querystring: {
     type: 'object',
     properties: {
+      /** @deprecated use accountId */
       memberId: { $ref: 'https://graasp.org/#/definitions/uuid' },
```
Comment (spaenleh) [inline src/services/item/plugins/app/appAction/schemas.ts:43]: Same comment here

File: src/services/item/plugins/app/appData/schemas.ts [hunk 3]
```diff
@@ -8,13 +6,15 @@ export default {
       properties: {
         id: { type: 'string' },
         account: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
+        /** deprecated use account - to support legacy apps */
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:9]:
```suggestion
        /** @deprecated use account - to support legacy apps */
```

File: src/services/item/plugins/app/appData/schemas.ts [hunk 4]
```diff
@@ -8,13 +6,15 @@ export default {
...
+        member: { $ref: 'https://graasp.org/accounts/#/definitions/account' },
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:10]: Here also add the JSONSchema deprecation

File: src/services/item/plugins/app/appData/schemas.ts [hunk 5]
```diff
@@ -8,13 +6,15 @@ export default {
...
         item: { $ref: 'https://graasp.org/items/#/definitions/item' },
         data: {
           type: 'object',
           additionalProperties: true,
         },
         type: { type: 'string' },
-        visibility: { type: 'string' }, // TODO: should we always return this
+        visibility: { type: 'string' },
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:17]: This should be an enum of values no ?

File: src/services/item/plugins/app/appData/schemas.ts [hunk 6]
```diff
@@ -32,6 +32,8 @@ const create = {
       data: { type: 'object', additionalProperties: true },
       type: { type: 'string', minLength: 3, maxLength: 25 },
       visibility: { type: 'string', enum: ['member', 'item'] },
+      /** @deprecated use accountId */
+      memberId: { $ref: 'https://graasp.org/#/definitions/uuid' },
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:36]: Same here for the deprecation

**Review by spaenleh — APPROVED** [review]
Looks good, maybe just add the deprecation as we saw for the JSONSchema 👍

File: src/services/item/plugins/app/docs/api.md [hunk 7]
```diff
@@ -27,8 +27,9 @@ Within Graasp, the apps are given some information by query string:
 App actions are analytic traces the app might save. They have the following structure:
 
 - `id`: the app action id
-- `memberId`: the member id related to the app action (default: current authenticated member id)
-- `itemId`: the item id corresponding to the current app
+- `account`: the member related to the app action (default: current authenticated member id).
+- `member`: use `account`
```
Comment (ReidyT) [inline src/services/item/plugins/app/docs/api.md:31]: Maybe explain more why we still need it?

File: src/services/item/plugins/app/docs/api.md [hunk 8]
```diff
@@ -59,11 +56,12 @@ App actions are analytic traces the app might save. They have the following stru
 App data are all data the app might save. They have the following structure:
 
 - `id`: the app data id
-- `memberId`: the member id related to the data (default: current authenticated member id)
-- `itemId`: the item id corresponding to the current app
+- `account`: the member related to the data (default: current authenticated member id)
+- `member`: use `account`
```
Comment (ReidyT) [inline src/services/item/plugins/app/docs/api.md:60]: same here

File: src/services/item/plugins/app/docs/api.md [hunk 9]
```diff
@@ -250,7 +244,8 @@ postMessage(
   - `context`: where the app is running (eg: `builder`, `explorer`, `standalone`, ...)
   - `itemId`: item id which corresponds to your app resource id
   - `lang`: language
-  - `memberId`: the current authenticated user using the app
+  - `accountId`: the current authenticated user using the app
+  - `memberId`: use `accountId`
```
Comment (ReidyT) [inline src/services/item/plugins/app/docs/api.md:248]: same here

**Review by ReidyT — APPROVED** [review]
LGTM 🦤

**Pull request merged**